exir/memory_planning.py

```diff
@@ -669,9 +669,8 @@ def get_node_tensor_specs(
         ]
 
 
-# Little bit hacky to check if the graph contains
-# XNNPACK delegate
-# Why?
+# Little bit hacky, because we are checking for xnnpack string in
+# backend name, to check if the graph contains XNNPACK delegate
```

Comment:
Can you mention this is for alignment. And open an issue and link it to this around the problem of delegates wanting to surface that they have alignment requirements for inputs

Reply:
To be sure this is not for alignment. It is for out-of-bounds access. I thought we did allow for alignment requirements from delegates elsewhere

Reply:
Oh yeah sorry not alignment, padding.